azure-pipelines/end-to-end-tests-dir/cli.ps1

```diff
@@ -9,3 +9,136 @@ Throw-IfNotFailed
 
 Run-Vcpkg -TestArgs ($commonArgs + @("install", "vcpkg-hello-world-1", "--fast")) # --fast is not a switch
 Throw-IfNotFailed
+
+if ($IsWindows) {
+    $warningText = 'Starting with the September 2023 release'
+
+    # build-external not tested
+    # ci not tested
+    # export not tested

```

Comment:
Any way we can test if `VCPKG_DEFAULT_TRIPLET` environmental variable is set? This should not emit a warning message.

Reply:
Done in 7166155f4b9509e3d1530a943295c5ebd33f4a0e !